### PR TITLE
chore(atomic): add aliases for main src folders

### PR DESCRIPTION
**Pull request by liviarett**
https://coveord.atlassian.net/browse/KIT-1790

### CODE REVIEW

**Review by ThibodeauJF — APPROVED** [review]
Pretty cool!

File: packages/atomic/stencil.config.ts [hunk 1]
```diff
@@ -19,6 +19,14 @@ import {generateAngularModuleDefinition as angularModule} from './stencil-plugin
 
 const isProduction = process.env.BUILD === 'production';
 
+const toAlias = ['global', 'images', 'utils', 'components'];
```
[the remark this answers is not in the document]
Comment (olamothe) [reply packages/atomic/stencil.config.ts:22]: We'll get `@components/search/searchbox` and `@components/insight/searchbox`, etc... 🤔

The separation seems pretty clean anyway, and it's not really making import path shorter ?

**Review by olamothe — APPROVED** [review]
💯

File: packages/atomic/src/components/search/atomic-refine-modal/atomic-refine-modal.tsx [hunk 2]
```diff
@@ -17,10 +17,10 @@ import {
...
   InitializeBindings,
-} from '../../../utils/initialization-utils';
+} from '@utils/initialization-utils';
 import CloseIcon from 'coveo-styleguide/resources/icons/svg/close.svg';
 import {SortDropdownOption} from '../atomic-search-interface/store';
-import SortIcon from '../../../images/sort.svg';
+import SortIcon from '@images/sort.svg';
 import {Button} from '../../common/button';
 import {Bindings} from '../atomic-search-interface/atomic-search-interface';
 import {BaseFacetElement} from '../facets/facet-common';
```
[the remark this answers is not in the document]
Comment (liviarett) [reply packages/atomic/src/components/search/atomic-refine-modal/atomic-refine-modal.tsx:26]: I'm actually not sure what to do with these, I thought that it was ok to keep them relative if they are within its own folder. if VSCode starts to automatically add them as absolute paths then let's do that in a separate PR! I'm saving this image on my laptop for future reference

**Review by mbenkirane-coveo — APPROVED** [review]

**Pull request merged**